File: src/main/java/com/cfgglobal/test/util/aws/UploadData.java
package com.cfgglobal.test.util.aws;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.io.Serializable;

@Data
@AllArgsConstructor
public class UploadData implements Serializable {

    private static final long serialVersionUID = -166575150661617870L;

    private String name;
    private byte[] data;
}

File: src/main/kotlin/com/github/leon/aws/s3/UploadData.kt
package com.github.leon.aws.s3


import java.io.Serializable


class UploadData : Serializable {

    private val name: String? = null
    private val data: ByteArray? = null

    companion object {

        private const val serialVersionUID = -166575150661617870L
    }
}
